egs/swbd/s5c/local/nnet3/run_lstm.sh

```diff
@@ -34,6 +34,7 @@ non_recurrent_projection_dim=256
 chunk_width=20
 chunk_left_context=40
 chunk_right_context=0
+ivector_interval=10
```

Comment:
Actually, might it be easier to, at the script level, hardcode this ivector_period to be the same as the one with which the iVectors were extracted on disk (if we extracted them using extract_ivectors_online)?  Then, instead of specifying it here as a config, you could just get it from the file that's written by extract_ivectors_online.sh (or whatever script we use, I forget the name).  That way, if the user wants to use a different ivector-period, they can change the run_ivector_common.sh script.  There is, IMO, no reason to have script-level support for using a different iVector period in the nnet than we have in the dumped iVectors.